### ISSUE TITLE
Shapeshifter gives graphics error on Android

### ISSUE THREAD

**Issue by Smashman4** [libretro/libretro-uae#568]
Hi,

Whenever I play a game that uses shapeshifter. It gives me an error saying "The Macintosh screen cannot be created. Either there is not enough graphics memory available, or the selected graphics mode is not displayable on your Amiga." I've tried to use the Amiga CD 32 ROM but still gives me that error. I'm not sure if this is an Android version issue or a core issue. But is there any way I can play games that use Shapeshifter on my Snapdragon 888 phone?

Thanks.

**Comment by sonninnos** [comment]
No, Shapeshifter is a MacOS emulator which requires RTG/Picasso96, which is not supported in this core at all, since it focuses on classic Amiga.

CD32 does not even have the required hardware for it, so there is no way to do that with it even if the core had the necessary building blocks.

**Comment by Smashman4** [comment]
Oh ok. I thought there was an option to enable it in the core. I guess I'll have to use minivmac if I want to try Mac games in retroarch.

**Comment by sonninnos** [comment]
Maybe some day, but currently not a priority with actual Amiga stuff.
